rslib/src/backend/mod.rs

```diff
@@ -72,12 +74,21 @@ pub fn init_backend(init_msg: &[u8]) -> result::Result<Backend, String> {
         };
 
     let tr = I18n::new(&input.preferred_langs);
+    let cert_str = input.custom_cert;
 
-    Ok(Backend::new(tr, input.server))
+    if cert_str.is_some() {
+        let cert = Certificate::from_pem(cert_str.unwrap().as_bytes());
+
+        if cert.is_ok() {
```

Comment:
Better to crash if user provided an invalid cert, instead of silently ignoring it.

Reply:
(to clarify: if the user is directly setting the env var, crashing makes sense. On mobile clients, some validation on the frontend would make sense)